src/guiapi/include/WindowMenuSpin.hpp

```diff
@@ -26,10 +26,12 @@ class IWiSpin : public AddSuper<WI_LABEL_t> {
 
     string_view_utf8 units;
     SpinType value;
+    size_t spin_val_width;
 
-    static Rect16::Width_t calculateExtensionWidth(const char *unit, size_t value_max_digits);
+    static Rect16::Width_t calculateExtensionWidth(size_t unit_len, char uchar, size_t value_max_digits);
```

Comment:
what is `uchar`?
Unit can have multiple characters like "mA"

Reply:
you should probably pass `size_t unit_padding` or `bool has_half_space_padding`

Reply:
multiple occurrence in WindowMenuSpin

Reply:
Picked from private.